.github/workflows/continuous_integration.yml

```diff
@@ -181,7 +181,7 @@ jobs:
         with:
           job-type: CI-GPU
           job-name: AutoGluon-Multimodal-Predictor
-          command: chmod +x ./.github/workflow_scripts/test_multimodal_predictor.sh && ./.github/workflow_scripts/test_multimodal_predictor.sh
+          command: chmod +x ./.github/workflow_scripts/test_multimodal_predictor.sh && ./.github/workflow_scripts/test_multimodal_predictor.sh --runsgpu
```

Comment:
Suggestion: The name "runsgpu" may not be clear in its meaning. It is advisable to consider renaming it to something more obvious, for example, "runonegpu".